develop/parachain-devs/runtime-development/FRAME/add-pallet-to-the-runtime.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Add Pallet to the Runtime
+description: Learn how to include and configure pallets in a Polkadot SDK-based runtime, from adding dependencies to implementing necessary traits.
+---
+
+# Add Pallet to the Runtime
+
+## Introduction
+
+As demonstrated in the [Build a Local Solochain](/tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain/){target=\_blank} guide, the [Polkadot SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank} provides a functional runtime that includes default FRAME development modules (pallets) to help you get started with building a custom blockchain.
+
+This article outlines the basic steps for adding a new pallet to the runtime of the node template. These steps apply when integrating a FRAME pallet into your runtime. However, each pallet has specific configuration requirements, such as the parameters and types needed to enable the pallet's functionality. In this guide, you'll learn how to add a pallet to the runtime and configure the settings specific to that pallet.
+
+The purpose of this article is to help you:
+
+- Learn how to update runtime dependencies to integrate a new pallet
+- Understand how to configure pallet-specific Rust traits to enable the pallet's functionality
+- Grasp the entire workflow of integrating a new pallet into your runtime
+
+## Configuring Runtime Dependencies
+
+For Rust programs, this configuration is defined in the `Cargo.toml` file, which specifies the settings and dependencies that control what gets compiled into the final binary. Since the Polkadot SDK runtime compiles to both a native binary (which includes standard Rust library functions) and a Wasm binary (which does not include the standard Rust library), the `runtime/Cargo.toml` file manages two key aspects:
+
+- The locations and versions of the pallets that are to be imported as dependencies for the runtime
+- The features in each pallet that should be enabled when compiling the native Rust binary. By enabling the standard (`std`) feature set from each pallet, you ensure that the runtime includes the functions, types, and primitives necessary for the native build, which are otherwise excluded when compiling the Wasm binary
+
+!!! note
+    For information about adding dependencies in `Cargo.toml` files, see [Dependencies](https://doc.rust-lang.org/cargo/guide/dependencies.html){target=\_blank} page in the Cargo documentation. For information about enabling and managing features from dependent packages, see [Features](https://doc.rust-lang.org/cargo/reference/features.html){target=\_blank} section in the Cargo documentation.
+
+## Dependencies for a New Pallet
+
+To add the dependencies for a new pallet to the runtime, you need to modify the `Cargo.toml` file by adding a new line into the `[workspace.dependencies]` section with the pallet you want to add. Ensure that you stick to the following format:
```

Comment:
You should not necessarily stick to this format, as it assumes that the pallet is uploaded to crates.io. This is probably the least used one. The most common use is the `git' or `path' field. We could mention the options or give an example.